README.rst

```diff
@@ -213,9 +213,11 @@ Here is a full example, below is the detail explanation:
     supervisor-email = zope@localhost
     supervisor-memmon-size = 1200MB
     supervisor-memmon-options = -a ${buildout:supervisor-memmon-size} -m ${buildout:supervisor-email}
+    supervisor-memmon-autostart = true
     supervisor-httpok-timeout = 40
     supervisor-httpok-options = -t ${buildout:supervisor-httpok-timeout} -m ${buildout:supervisor-email}
     supervisor-httpok-view =
+    supervisor-httpok-autostart = false
```

Comment:
Shouldn't we use the defaults in the `README` examples? I.e. `supervisor-httpok-autostart = true`?